.github/workflows/deploy.yml

```diff
@@ -22,9 +22,11 @@ jobs:
       E2E_PAT: ${{secrets.E2E_PAT}}
       RSA256_PUBLIC_KEY_BASE64: ${{secrets.RSA256_PUBLIC_KEY_BASE64}}
       GOV_NOTIFY_API_KEY: ${{secrets.GOV_NOTIFY_API_KEY}}
-  paketo_build:
-    permissions:
-      packages: write
-    uses: ./.github/workflows/package.yml
-    with:
-      version_to_build: $(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+    paketo_build:
```

Comment:
Why the indentation change?  Isn't this on the same level as test_and_deploy?

Reply:
Interesting...this doesn't show up like this in the IDE 🤔 
![image](https://github.com/communitiesuk/funding-service-design-notification/assets/125875751/ed0af2ba-fb86-4bdf-ab0c-19c0e516fa8d)

Reply:
Quick commit _message_ edit has resolved this. No idea why 🤷